bridgetown-core/lib/bridgetown-core/commands/configure.rb

```diff
@@ -38,10 +38,18 @@ def self.source_root
       protected
 
       def configure(configuration)
-        configuration_file = find_in_source_paths("#{configuration}.rb")
+        if configuration.include?(":")
+          configuration_filename = configuration.split(":").first
+          configuration_option = configuration.split(":").last.split.first
```

Comment:
The final `split.first` is to ignore any text _after_ the option. It seems if you run multiple configurations in a single command the second command shows up?

Eg. `bridgetown configure syntax-theme:github vercel`

`configuration_option` will be `github vercel` without the final `split.first`

It does seem odd to me that this occurs as I thought the `args.each do |configuration|` line above would already sort this?

Reply:
Yeah that is quite weird; although I just tried it and it seems to be working as expected for me 🤷‍♂️. i.e. the second configuration option is not passed through to the `configure` method. Could you try again to confirm this is still happening for you?

Reply:
```suggestion
      def configure(configuration)
        filename, *options = configuration.split(":")
        file = find_in_source_paths("#{configuration_filename}.rb")

        inside(New.created_site_dir || Dir.pwd) do
          @templates_dir = File.expand_path("../configurations/#{filename}", __dir__)
          @options = options
          apply file, verbose: false
        end
      end
```